public_html/about.php

```diff
@@ -40,6 +42,12 @@
             $contributors_html .= '</a>';
         $contributors_html .= '</h6>';
     }
+    // Address
+    if(array_key_exists('address', $c) && array_key_exists('full_name', $c)){
```

Comment:
Should be `array_key_exists('location', $c)`

Reply:
Good catch, totally forgot about that